packages/client/src/Loc.ts

```diff
@@ -914,6 +947,39 @@ export class LegalOfficerEditableRequestCommands {
         });
         return await this.request.refresh() as EditableRequest;
     }
+
+    async addLink(params: AddLinkParams): Promise<EditableRequest> {
+        await this.client.addLink({
+            locId: this.locId,
+            ...params
+        });
+        return await this.request.refresh() as EditableRequest;
+    }
+
+    async deleteLink(params: DeleteLinkParams): Promise<EditableRequest> {
+        await this.client.deleteLink({
+            locId: this.locId,
+            ...params
+        })
+        return await this.request.refresh() as EditableRequest
+    }
+
+    async publishLink(parameters: { target: UUID } & BlockchainSubmissionParams): Promise<EditableRequest> {
+        const link = this.request.data().links.find(link => link.target === parameters.target.toString());
+        if(!link) {
+            throw new Error("Link was not found");
+        }
+        await this.client.publishLink({
+            locId: this.locId,
+            link: {
+                id: parameters.target,
+                nature: hashString(link.nature),
+            },
+            signer: parameters.signer,
+            callback: parameters.callback,
+        });
+        return await this.request.refresh() as EditableRequest;
+    }
 }
```

Comment:
I don't think we need to define `publishLink()` at `EditableRequest` level. From what I can see, it is anyway overridden in `LegalOfficerOpenRequestCommands`, and it makes no sense to have it on a draft request.
Similarly, in user SDK, `publishMetadata` and `publishFile` only exist on `openLoc`.

Reply:
You're right, I forgot to remove the function from user API. Will do that now.